tools/node.js

```diff
@@ -120,7 +121,8 @@ exports.minify = function(files, options) {
     }
     if (options.outSourceMap || options.sourceMapInline) {
         output.source_map = UglifyJS.SourceMap({
-            file: options.outSourceMap,
+            // for backward compatible, use outFileName first, then deduce from outSourceMap (remove .map), then outSourceMap
```

Comment:
Since the change is not strictly backwards compatible could you reword the comment as follows?
```
// prefer outFileName, otherwise use outSourceMap without .map suffix
```